### PR TITLE
Dedicated notifications type

### PR DESCRIPTION
**Pull request by e-kolpakov**
**Background:** Group Project XBlock sends notifications for file upload, grades posted and stage open/close dates. Group Project v2 sends the same types of messages, but uses different navigation approach, so the URLs in messages need to be different. For this to work, GP v2 has to send messages of unique types. 

**Description:** This PR switches GP v2 to use dedicated message types

**Dependencies:** This PR is one of the three that fix notifications in Group Project v2 XBlock; the other two are:
* https://github.com/edx-solutions/edx-platform/pull/517 - adds URLResolvers
* https://github.com/edx/edx-notifications/pull/162 - registers new message types

**Related:** [YONK-146](https://openedx.atlassian.net/browse/YONK-146) was caused by changing URL resolvers for GP v1. Should not be affected by this change.

**Testing instructions:** see testing instructions on https://github.com/edx-solutions/edx-platform/pull/517

### CODE REVIEW

File: group_project_v2/notifications.py [hunk 1]
```diff
@@ -119,13 +137,10 @@ def on_before_studio_delete(self, course_id, services):  # pylint: disable=unuse
 
         notifications_service = services.get('notifications')
         if notifications_service:
-            # If we are being delete, then we should remove any NotificationTimers that
+            # If stage are being deleted, then it should remove any NotificationTimers that
```
Comment (Kelketek) [inline group_project_v2/notifications.py]: "If stages are", or "If a stage is"

**Comment by Kelketek** [comment]
@e-kolpakov Can you check the hashes in your requirements on this branch? The build doesn't run.

**Comment by e-kolpakov** [comment]
@Kelketek sure, I'm trying to make it run again for quite some time. This is the third missing hash already.

**Comment by Kelketek** [comment]
:+1:

**Pull request merged**